Bot/Storage.py

```diff
@@ -1,30 +1,55 @@
-import datetime
+from datetime import datetime
+import json
+import os
 from typing import Optional
-
+from logging import Logger
 
 class Storage:
-    def __init__(self) -> None:
+    def __init__(self, logger : Logger) -> None:
         """Stores the information for each user"""
         # Load the shelve db if possible
-        self.storage = {}
+        if os.path.isfile(f"./database/storage.json"):
```

Comment:
```python
  BASE_DIR = os.path.dirname(os.path.abspath(__file__))
  ...
  path = os.path.join(BASE_DIR,"database","storage.json")
```

Filepath can be improved using the snippet above instead of `./` as the location may vary based on the directory where the command was run.